DotNetCoreMasters/DotNetCoreMasters/Startup.cs

```diff
@@ -14,13 +19,21 @@ namespace DotNetCoreMasters
 {
     public class Startup
     {
+        IConfiguration Configuration { get; }
+
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
         // This method gets called by the runtime. Use this method to add services to the container.
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers();
             services.AddItemService();
+            services.Configure<Keys.Authentication>(Configuration.GetSection(nameof(Authentication)));
 
+            services.AddMvc().AddMvcOptions(options => options.Filters.Add(new ShowElapseTimeAttribute()));
```

Comment:
need to add the filter thru the services.AddControllers. Exact same syntax just need to re-use the existing AddControllers call